src/Stratis.Bitcoin.Tests/Builder/FullNodeBuilderExtensionsTest.cs

```diff
@@ -36,17 +36,16 @@ public void UseNodeSettingsConfiguresNodeBuilderWithNodeSettings()
             Assert.Equal(NodeSettings.Default().ConfigurationFile, this.fullNodeBuilder.NodeSettings.ConfigurationFile);
             Assert.Equal(NodeSettings.Default().DataDir, this.fullNodeBuilder.NodeSettings.DataDir);
             Assert.NotNull(this.fullNodeBuilder.Network);
-            Assert.Equal(NodeSettings.Default().GetNetwork(), this.fullNodeBuilder.Network);
+            Assert.Equal(NodeSettings.Default().Network, this.fullNodeBuilder.Network);
             Assert.Single(this.serviceCollectionDelegates);
         }
 
         [Fact]
         public void UseDefaultNodeSettingsConfiguresNodeBuilderWithDefaultSettings()
         {
-            var nodeSettings = NodeSettings.Default();
+            var nodeSettings = new NodeSettings();
```

Comment:
Why the change? You could have `NodeSettings.Default(Network.TestNet);`

Reply:
I was making some changes to the arguments and then settled on not changing anything. I can revert that to the Default method although it would have the same effect. Using Network.Testnet would make this test case fail due to the `Assert.Equal(Network.Main, this.fullNodeBuilder.Network);`.

Reply:
I will change all 3 methods in that class to use the Default method.